ansible/roles/openregister_config/templates/openregister-config.yaml.j2

```diff
@@ -58,6 +58,10 @@ downloadConfigs: true
 
 server:
   registerDefaultExceptionMappers: false
+  requestLog:
+    type: classic
```

Comment:
I don't entirely understand why it's needed but without it it fails to log requests /shrug